bool vibrotactile
bool skin_stretch
---

